plugins/nvme/nvme.quirk

```diff
@@ -43,3 +43,7 @@ Flags = unsigned-payload
 # Western Digital
 [NVME\VEN_101C]
 Flags = signed-payload
+
+# Solidigm
+[NVME\VEN_025E&DEV_F1AB]
```

Comment:
Any more you want to add? i.e. any variants or future models?

Reply:
No, just this specific device